### PR TITLE
#1 strategy

### PR DESCRIPTION
**Pull request by liliyafedyushko**

### CODE REVIEW

File: strategy/src/main/java/users/UserEmailRegisterMobileAppLogIn.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package users;
+
+import strategies.login.LogInWithMobileApp;
+import strategies.register.EmailRegistrationStrategy;
+
+public class UserEmailRegisterMobileAppLogIn extends User{
```
Comment (mykola-veremeichyk) [inline strategy/src/main/java/users/UserEmailRegisterMobileAppLogIn.java]: Add whitespace in this line ;)

File: strategy/src/main/java/users/UserOauthRegisterWebsiteLogIn.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package users;
+
+import strategies.login.LogInWithWebsite;
+import strategies.register.OAuthRegistrationStrategy;
+
+public class UserOauthRegisterWebsiteLogIn extends User{
```
Comment (mykola-veremeichyk) [inline strategy/src/main/java/users/UserOauthRegisterWebsiteLogIn.java]: Add whitespace, please ;)

File: strategy/src/main/java/strategies/login/LogInWithWebsite.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package strategies.login;
+
+import strategies.LogInStrategy;
+
+public class LogInWithWebsite implements LogInStrategy {
+
+  @Override
+  public void logIn() {
+
```
Comment (mykola-veremeichyk) [inline strategy/src/main/java/strategies/login/LogInWithWebsite.java:9]: I see you are using the empty line after the method declaration. Do you use this kind of practice on your project?

Comment (liliyafedyushko) [reply strategy/src/main/java/strategies/login/LogInWithWebsite.java:9]: no, not intentionally)

**Pull request merged**